iocage/cli/activate.py

```diff
@@ -23,20 +23,40 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """activate module for the cli."""
 import click
+import libzfs
 
-import iocage.lib.ioc_common as ioc_common
-import iocage.lib.iocage as ioc
+import iocage.lib.Datasets
+import iocage.lib.Logger
 
 __rootcmd__ = True
 
 
 @click.command(name="activate", help="Set a zpool active for iocage usage.")
 @click.argument("zpool")
-def cli(zpool):
+@click.option("--log-level", "-d", default=None)
+@click.option("--mountpoint", "-m", default="/iocage")
+def cli(zpool, log_level, mountpoint):
     """Calls ZFS set to change the property org.freebsd.ioc:active to yes."""
-    ioc.IOCage(activate=True).activate(zpool)
+    logger = iocage.lib.Logger.Logger(print_level=log_level)
+    zfs = libzfs.ZFS(history=True, history_prefix="<iocage>")
```

Comment:
I don't immediately see the issue of the cli using the library helper.
Both are doing the exactly same thing, hence why I asked.